### PR TITLE
Update diversity toturial

### PR DESCRIPTION
**Pull request by paulzierep**
After fixing the linting and links I intend to properly set this tutorial up

### CODE REVIEW

File: topics/microbiome/tutorials/diversity/workflows/workflow.ga [hunk 1]
```diff
@@ -2,6 +2,13 @@
     "a_galaxy_workflow": "true",
     "annotation": "",
     "comments": [],
+    "creator": [
```
Comment (bebatut) [inline topics/microbiome/tutorials/diversity/workflows/workflow.ga:5]: Could you also add Sophia there? Thanks

Comment (paulzierep) [reply topics/microbiome/tutorials/diversity/workflows/workflow.ga:5]: sure !

**Review by shiltemann — DISMISSED** [review]
Thanks @paulzierep! looks good from a technical point of view :+1:

**Comment by paulzierep** [comment]
thanks @shiltemann - I would indeed set it to not draft, as I would like to record a video in the afternoon, can you approve the PR - if test work out - so I can merge ?

**Review by shiltemann — APPROVED** [review]

**Pull request merged**